agimus_controller/agimus_controller/warm_start_factory.py

```diff
@@ -0,0 +1,13 @@
+from .warm_start_base import WarmStartBase
+
+
+class WarmStartFactory(object):
+    def __init__(self) -> None:
+        pass
+
+    def create_warm_start(self, name: str) -> WarmStartBase:
+        if name == "from_previous_solution":
+            return None
+
+        if name == "from_diffusion_model":
+            return None
```

Comment:
Maybe having an hybrid, like :"from diffusion model but from previous solution as well" ? I don't really know how this factory works, but it looks heavy to use imo